### ISSUE TITLE
Applications have failed: log_mgr

### ISSUE THREAD

**Issue by drewelliott** [srl-labs/netbox-nrx-clab#7]
I have installed and run this on 5 different machines, all running ubuntu 24.04 and I have run into the same issue on each: 

(I am showing a single container, but this occurs on all 6 of the srl containers)

During clab deploy:
```
ERRO[0068] failed to run postdeploy task for node syd1-pd1-l1: command execution error:Error: Error 'Application failure before commit':
    [FailedPrecondition] Applications have failed: log_mgr
```

*here is the log from that container*
[log.txt](https://github.com/user-attachments/files/16629651/log.txt)

**Comment by hellt** [comment]
try with srlinux 24.7.1

**Comment by drewelliott** [comment]
This is the version I am currently using

```
--{ [FAILED] + running }--[  ]--
A:syd1-pd1-l1# show version
----------------------------------------------------------------------------------------------------------
Hostname             : syd1-pd1-l1
Chassis Type         : 7220 IXR-D3L
Part Number          : Sim Part No.
Serial Number        : Sim Serial No.
System HW MAC Address: 1A:DA:01:FF:00:00
OS                   : SR Linux
Software Version     : v24.7.1
Build Number         : 330-g38f237abfe
Architecture         : x86_64
Last Booted          : 2024-08-13T22:48:47.173Z
Total Memory         : 257554660 kB
Free Memory          : 245111707 kB
----------------------------------------------------------------------------------------------------------

--{ [FAILED] + running }--[  ]--
A:syd1-pd1-l1#```

**Comment by tardoe** [comment]
Thanks for the report and logs @drewelliott. I'll work with the team to figure out what's the issue here. I'm pretty confident containerlab + SRL will work fine on 22.04.x LTS. We're investigating in the backend any Containerlab/SRL issues on 24.04.

**Comment by hellt** [comment]
This is an srl+ubuntu23+ issue. We can close this issue here as we need to work on the fix on the srl side

**Comment by hellt** [comment]
@drewelliott apply this workaround (on the host)

```
sudo ln -s /etc/apparmor.d/usr.sbin.rsyslogd /etc/apparmor.d/disable/
sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.rsyslogd
```

this will disable apparmor check for the rsyslog daemon. Unfortunately, I do not yet know when this will be fixed in the SRL itself, so the workaround is the only way forward when Ubuntu 23+ systems are concerned

**Comment by drewelliott** [comment]
this workaround corrected my issue on every ubuntu 24.04 host - closing
